src/components/ExperimentDetail.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useState } from 'react'
+// import { Link } from 'react-router-dom'
+import { Container, Row, Col, Alert, Card, Progress, CardBody, CardText, FormGroup, Label, Input, CustomInput, DropdownToggle, DropdownMenu, DropdownItem, UncontrolledButtonDropdown, Nav, NavItem, NavLink, TabContent, TabPane } from "reactstrap";
+import "../static/css/Experiments.css";
+import classnames from 'classnames';
+ 
+
+const ExperimentDetail = () => {
+
+    const data = [
+        { id: "1", emojis: "🚀💸", name:"Up Only WBTC" , contract: "OxSalo12345", assetPrice: "1.00", currency: "$", totalAssets: "1000", totalAUM: "10000", baseAsset: "WBTC"},
+      ];
+
+    const [activeTab, setActiveTab] = useState('1');
+
+    const toggle = (tab: any) => {
+        if(activeTab !== tab) setActiveTab(tab);
+    }
+
+    const navItems = ['Details', 'Wallet', 'Contracts']
+
+    return (
+        <Container className="experiments-container container h-100">
+            <Row className="row h-100 justify-content-center align-items-center">
```

Comment:
I see the tabs are inconsistent, it's 4 tabs here whereas it's 2 tabs everywhere. Could we please stick to 2 tabs only?

Reply:
fixed!